internal/tfengine/tfengine.go

```diff
@@ -225,6 +229,20 @@ func dumpTemplate(conf *Config, pwd, cacheDir, outputPath string, ti *templateIn
 	return nil
 }
 
+func addGitIgnore(path string) error {
+	data := []byte(`# Local .terraform directories
+**/.terraform/*
+
+# .tfstate files
+*.tfstate
+*.tfstate.*
+`)
```

Comment:
Have you considered instead using this https://github.com/github/gitignore/blob/master/Terraform.gitignore?

Reply:
Ignoring .terraform. dirs and state files should cover all temporary files that are generated by DPT operations currently. And we do need terraform.tfvars files to be checked in as they are generated and used by DPT. If users end up modifying their generated Terraform configs, they can add more things in the .gitignore file.

Reply:
Makes sense to me.